espnet2/asr_transducer/error_calculator.py

```diff
@@ -39,8 +41,9 @@ def __init__(
         self.beam_search = BeamSearchTransducer(
             decoder=decoder,
             joint_network=joint_network,
-            beam_size=1,
-            search_type="default",
+            beam_size=2,
+            search_type="maes",
```

Comment:
Is it OK to change it from `default` to `maes`?
What is the intention of this change?

Reply:
> Is it OK to change it from default to maes?

It is, it won't make a difference for the reporting.

> What is the intention of this change?

To control the number of emitted symbols at each timestep in both training and validation.
With k2, we can end-up with a big performance degradation if we enforce an unrestricted decoding without regards to the training condition/parameters (See. https://arxiv.org/pdf/2211.00484.pdf). With `maes`, it'll be "stable" under all conditions.

Also, in some cases you can end-up with a really long looping procedure with `default` (e.g.: during early stages with some architecture such as MEGA or if there is a training/decoding condition mismatch as I said). It won't happen with other decoding algorithms.

Reply:
Thanks for the clarification.
I also encountered the issue of the number of emitted symbols, and this change sounds good to me.
Can you leave this discussion somewhere in the source code?

Reply:
Done but I'm not sure it's in a right place nor in a right form..!